Common/src/main/java/com/blamejared/crafttweaker/api/zencode/impl/loader/ScriptRun.java

```diff
@@ -101,127 +89,81 @@ public void run() throws Exception {
-        
-        scriptingEngine.registerNativeProvided(crafttweakerModule);
-        modules.add(crafttweakerModule);
-        
-        final HashSet<String> rootPackages = new HashSet<>(CraftTweakerRegistry.getRootPackages());
-        rootPackages.remove(CraftTweakerConstants.MOD_ID);
-        for(String rootPackage : rootPackages) {
-            final JavaNativeModule module = createModule(bep, rootPackage, rootPackage, nativeConverterBuilder, crafttweakerModule);
-            scriptingEngine.registerNativeProvided(module);
-            modules.add(module);
-        }
+    private void writeFormattedFiles(final SemanticModule scripts) {
         
+        final List<HighLevelDefinition> all = scripts.definitions.getAll();
+        final ScriptFormattingSettings.Builder builder = new ScriptFormattingSettings.Builder();
+        final FileFormatter formatter = new FileFormatter(builder.build());
+        final Path root = Paths.get("scriptsFormatted");
         
-        final JavaNativeModule expModule = createModule(bep, "expansions", "", nativeConverterBuilder, modules.toArray(new JavaNativeModule[0]));
-        for(List<Class<?>> expansionList : CraftTweakerRegistry.getExpansions().values()) {
-            for(Class<?> expansionClass : expansionList) {
-                expModule.addClass(expansionClass);
-            }
-        }
-        scriptingEngine.registerNativeProvided(expModule);
-        
-        nativeConverterBuilder.headerConverter.reinitializeAllLazyValues();
-    }
-    
-    private JavaNativeModule createModule(IgnorePrefixCasingBracketParser bep, String moduleName, String basePackage, JavaNativeConverterBuilder nativeConverterBuilder, JavaNativeModule... dependencies) {
-        
-        JavaNativeModule module = scriptingEngine.createNativeModule(moduleName, basePackage, dependencies, nativeConverterBuilder);
-        
-        
-        for(ValidatedEscapableBracketParser bracketResolver : CraftTweakerRegistry.getBracketResolvers(moduleName, scriptingEngine, module)) {
-            bep.register(bracketResolver.getName(), bracketResolver);
-        }
-        module.registerBEP(bep);
-        for(Class<?> aClass : CraftTweakerRegistry.getGlobalsInPackage(moduleName)) {
-            module.addGlobals(aClass);
-        }
-        for(Class<?> aClass : CraftTweakerRegistry.getClassesInPackage(moduleName)) {
-            module.addClass(aClass);
-        }
-        
-        return module;
-    }
-    
-    private void writeFormattedFiles(SemanticModule scripts) {
-        
-        List<HighLevelDefinition> all = scripts.definitions.getAll();
-        ScriptFormattingSettings.Builder builder = new ScriptFormattingSettings.Builder();
-        FileFormatter formatter = new FileFormatter(builder.build());
-        List<ScriptBlock> blocks = scripts.scripts;
-        for(ScriptBlock block : blocks) {
+        scripts.scripts.forEach(block -> {
```

Comment:
And... why did you convert that foreach to a forEach?
Or was that applied via codestyle rules?